tests/dapr_tests.mk

```diff
@@ -318,6 +320,11 @@ setup-test-components: setup-app-configurations
 	$(KUBECTL) apply -f ./tests/config/app_topic_subscription_routing.yaml --namespace $(DAPR_TEST_NAMESPACE)
 	$(KUBECTL) apply -f ./tests/config/app_topic_subscription_routing_grpc.yaml --namespace $(DAPR_TEST_NAMESPACE)
 	$(KUBECTL) apply -f ./tests/config/app_pubsub_routing.yaml --namespace $(DAPR_TEST_NAMESPACE)
+	$(KUBECTL) apply -f ./tests/config/app_resiliency.yaml --namespace $(DAPR_TEST_NAMESPACE)
+	$(KUBECTL) apply -f ./tests/config/resiliency.yaml --namespace $(DAPR_TEST_NAMESPACE)
+	$(KUBECTL) apply -f ./tests/config/resiliency_kafka_bindings.yaml --namespace $(DAPR_TEST_NAMESPACE)
```

Comment:
As you might have seen, we have the E2E tests to be configurable via DAPR_TEST_PUBSUB, DAPR_TEST_STATE_STORE and DAPR_TEST_QUERY_STATE_STORE. Can you make sure the redis resiliency test works with ASB too? This is because we use ASB for E2E on AKS.

Reply:
Added the file to see if it still works here. Still working on getting a local version of it running with ASB.

Reply:
Slight change in the config and we're good to go.